src/arrayMethodSort.js

```diff
@@ -4,9 +4,17 @@
  * Implement method Sort
  */
 function applyCustomSort() {
-  [].__proto__.sort2 = function(compareFunction) {
+  [].__proto__.sort2 = function (callback = (a, b) => String(a).localeCompare(String(b))) {
     // write code here
-  };
+    for (let i = 0; i < this.length; i++) {
+      for (let j = i + 1; j < this.length; j++) {
+        if (callback(this[i], this[j]) > 0) {
+          [this[i], this[j]] = [this[j], this[i]]
```

Comment:
When swapping elements, it's a good practice to create a temporary variable for one of the elements. This can help prevent potential issues in environments that don't support array destructuring.